src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunctionSet.cs

```diff
@@ -0,0 +1,333 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.PowerFx.Core.Texl.Builtins;
+using Microsoft.PowerFx.Core.Utils;
+
+namespace Microsoft.PowerFx.Core.Functions
+{
+    internal class TexlFunctionSet
+    {
+        private Dictionary<string, List<TexlFunction>> _functions;
```

Comment:
>string [](http://example.com/codeflow?start=27&length=6)

nit: can we comment what the key is here? 
This looks like function name to a ordered list of functions with that name.